lib/src/fields/form_builder_checkbox.dart

```diff
@@ -68,12 +68,12 @@ class FormBuilderCheckbox extends FormBuilderField<bool> {
     //From Super
     Key key,
     @required String name,
-    FormFieldValidator validator,
+    FormFieldValidator<bool> validator,
     bool initialValue,
     bool readOnly = false,
     InputDecoration decoration = const InputDecoration(),
-    ValueChanged onChanged,
-    ValueTransformer valueTransformer,
+    ValueChanged<bool> onChanged,
+    ValueTransformer<bool> valueTransformer,
     bool enabled = true,
     FormFieldSetter onSaved,
```

Comment:
Anytime you see `FormFieldSetter` and `FormFieldState` with out a `<type>`, that is an opportunity to add type safety.  In this case:  `FormFieldSetter<bool> onSaved`.

Reply:
Oh yeah I forgot that one🤦‍♂️

Reply:
I recommend doing a quick search as I discovered a couple.  I love type safety! 🤓

Reply:
Yes there are still a lot of them! Me too! 😂